sys/key_value.js

```diff
@@ -194,6 +195,32 @@ class KVBucket {
             hyper.log('error/kv/putRevision', error);
             return { status: 400 };
         });
+
+        if (req.query.ignore_duplicates) {
+            return hyper.get({
+                uri: new URI([rp.domain, 'sys', 'key_value', rp.bucket, rp.key])
+            })
+            .then((oldContent) => {
+                if (stringify(req.body) === stringify(oldContent.body) &&
+                    (!req.headers['content-type']
```

Comment:
Nit: let's indent this one more level so as not to confuse it with the statements inside the `if` block